plugins/periskop-backend/package.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
i think the build problem could be related to the roles change that you just missed - see this

https://github.com/backstage/backstage/pull/9254/files#diff-759b40a175557d29bac221ebffab3ed340170d4180937dd475540db58b3256bdR14

which shows the change needed specifically for a backend plugin

Reply:
Ah, I wasn't aware of this indeed. Will apply.